reaktive/src/commonTest/kotlin/com/badoo/reaktive/testutils/TestObservable.kt

```diff
@@ -1,26 +1,38 @@
 package com.badoo.reaktive.testutils
 
+import com.badoo.reaktive.disposable.Disposable
 import com.badoo.reaktive.observable.Observable
 import com.badoo.reaktive.observable.ObservableCallbacks
 import com.badoo.reaktive.observable.ObservableObserver
+import com.badoo.reaktive.utils.atomicreference.AtomicReference
+import com.badoo.reaktive.utils.atomicreference.update
 
-class TestObservable<T> : Observable<T>, ObservableCallbacks<T> {
+class TestObservable<T> : Observable<T>, ObservableCallbacks<T>, Disposable {
 
-    private lateinit var observer: ObservableObserver<T>
+    private val observersRef: AtomicReference<List<ObservableObserver<T>>> = AtomicReference(emptyList(), true)
```

Comment:
Do you want to group `private` and `public` properties together?

Reply:
Will rename `xxxRef` to `_xxx' as they are backing fields